stylelint-polaris/index.js

```diff
@@ -162,7 +164,6 @@ const stylelintPolarisCoverageOptions = {
         'grid-template-rows',
         'grid-template-columns',
         'grid-area',
-        'display',
```

Comment:
Why are `position` and `display` removed? It looks like the replacements don't cover the same scope. I thought having these set to warnings gives us insight on potential/future layout components.

Reply:
Good call, I've removed those changes as they were incomplete explorations of addressing some of the things we need to iron out in #7827 💯